### ISSUE TITLE
Show authors on the page

### ISSUE THREAD

**Issue by rijkvanzanten** [directus/docs#188]
With more and more contributors to the docs, it would be nice to show a little list of people who worked on a page on the bottom of every page. Kinda like how github does it:

<img width="467" alt="CleanShot 2022-10-24 at 12 01 02@2x" src="https://user-images.githubusercontent.com/9141017/197572286-3b98d9e4-50a9-4046-805b-41e3c77ad334.png">

**Comment by benhaynes** [comment]
Absolutely! And this would probably motivate more people to contribute.
